### PR TITLE
fix(ui): Icons inside tooltips

### PR DESCRIPTION
**Pull request by vuluongj20**
Fixing icon with inaccessible/invisible colors inside tooltips. See comments below for screenshots.

### CODE REVIEW

File: static/app/components/clipboardTooltip.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ function ClipboardTooltip({title, onSuccess, ...props}: Props) {
           <TextOverflow>{title}</TextOverflow>
           <Clipboard value={title} onSuccess={onSuccess}>
             <TooltipClipboardIconWrapper>
-              <IconCopy size="xs" color="white" aria-label={t('Copy to clipboard')} />
+              <IconCopy size="xs" aria-label={t('Copy to clipboard')} />
```
Comment (vuluongj20) [inline static/app/components/clipboardTooltip.tsx:29]: Before:
Screen Shot 2021-12-01 at 3 27 24 PM

After:
Screen Shot 2021-12-01 at 3 27 35 PM

File: static/app/components/seenByList.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const SeenByList = ({
       />
       <IconWrapper iconPosition={iconPosition}>
         <Tooltip title={iconTooltip}>
-          <IconShow size="sm" color="gray200" />
+          <IconShow size="sm" color="subText" />
```
Comment (vuluongj20) [inline static/app/components/seenByList.tsx:67]: Before
seenByList-b

After:
seenByList-a

Comment (evanpurkhiser) [reply static/app/components/seenByList.tsx:67]: What's it look like in light mode?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/seenByList.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/seenByList.tsx:67]: Screen Shot 2021-12-01 at 4 15 38 PM

**Pull request merged**